apps/etterlatte-behandling/src/main/kotlin/behandling/generellbehandling/GenerellBehandlingService.kt

```diff
@@ -146,11 +146,12 @@ class GenerellBehandlingService(
         generellBehandling: GenerellBehandling,
         saksbehandler: Saksbehandler,
     ) {
-        val hentetBehandling = hentBehandlingMedId(generellBehandling.id) ?: throw GenerellIkkeFunnetException()
-        if (!hentetBehandling.kanEndres()) {
-            throw FeilStatusGenerellBehandling(
-                "Behandlingen må ha status opprettet eller returnert, hadde ${generellBehandling.status} id: ${generellBehandling.id}",
-            )
+        with(hentBehandlingMedId(generellBehandling.id) ?: throw GenerellIkkeFunnetException()) {
+            if (!kanEndres()) {
+                throw FeilStatusGenerellBehandling(
+                    "Behandlingen må ha status opprettet eller returnert, hadde $status id: $id",
```

Comment:
Burde kanskje opprettet og returnert peke på faktiske enum-konstanter i feilmeldingen her? Slik at det blir riktig case osv.

Reply:
Forskjellen er at vi nå viser statusen til behandlingen vi har hentet fra DB, ikke den vi får inn fra routen. Feilmeldingen var misvisende, for det var forskjellige statuser på dem.